src/Pages/form/Validation/validations.ts

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+interface NIHSSProps {
+  levelOfConsciousness: string;
+  twoQuestions: string;
+  twoCommands: string;
+  bestGaze: string;
+  bestVisual: string;
+  facialPalsy: string;
+  bestMotorLeftArm: string;
+  bestMotorRightArm: string;
+  bestMotorLeftLeg: string;
+  bestMotorRightLeg: string;
+  limbAtaxia: string;
+  sensory: string;
+  bestLanguageAphasia: string;
+  dysarthria: string;
+  extinctionOrNeglect: string;
+}
+
+interface ValidateProps {
+  NIHSS: NIHSSProps;
+}
+
+const validate = (values: ValidateProps) => {
+  const errors = {
+    NIHSS: {
+      levelOfConsciousness: "",
+      twoQuestions: "",
+      twoCommands: "",
+      bestGaze: "",
+      bestVisual: "",
+      facialPalsy: "",
+      bestMotorLeftArm: "",
+      bestMotorRightArm: "",
+      bestMotorLeftLeg: "",
+      bestMotorRightLeg: "",
+      limbAtaxia: "",
+      sensory: "",
+      bestLanguageAphasia: "",
+      dysarthria: "",
+      extinctionOrNeglect: "",
+    },
+  };
+
+  if (
+    values.NIHSS.levelOfConsciousness === null ||
```

Comment:
As I see, there are many line same as this. So, it should be the group to function like this `checkBoxCondition(values)` (this is an example function name). And you can call this function in if else condition like this `if(checkBoxCondition(values.NIHSS.levelOfConsciousness))`.